features/lorawan/lorastack/mac/LoRaMacCommand.cpp

```diff
@@ -167,6 +167,13 @@ lorawan_status_t LoRaMacCommand::process_mac_commands(const uint8_t *payload, ui
                                                    &link_adr_nbtrans,
                                                    &link_adr_nb_bytes_pasred);
 
+                // If nothing was consumed, we have a malformed packet at our hand
+                // we bin everything and return. link_adr_nb_bytes_pasred being 0 is
+                // a magic identifier letting us know that there are payload inconsistencies
+                if (link_adr_nb_bytes_pasred == 0) {
```

Comment:
```suggestion
                if (link_adr_nb_bytes_parsed == 0) {
```

Reply:
I think you have a consistent typo 😄

Reply:
Am I missing something here @cmonr ? Your suggested change looks exactly the same as original.

Reply:
Bytes 'parsed' vs 'pasred'

Reply:
@cmonr Hawk eyes mate :)